src/game/common/globaldata.cpp

```diff
@@ -1039,7 +1039,8 @@ GlobalData::GlobalData()
 #ifdef PLATFORM_WINDOWS
     m_doubleClickTime = GetDoubleClickTime();
 #else
-// TODO, probably based on whatever crossplatform event framework we use.
+    // TODO, probably based on whatever crossplatform event framework we use. For now use a constant (1000ms)
+    m_doubleClickTime = 1000;
```

Comment:
Is this a reasonable duration? Seems quite long for a double click. Seems Grandpa friendly tho.